components/map.jsx

```diff
@@ -1,70 +1,54 @@
 var React = require('react');
-
+var mapboxId = process.env.MAPBOX_MAP_ID || 'alicoding.ldmhe4f3';
```

Comment:
err it was the access token i thought should be moved here, since we're setting it globally on `L.mapbox.accessToken`--not the map id... Though *defaulting* the `mapId` property to `alicoding.ldmhe4f3` probably isn't a bad idea, as it makes it super easy to plop a map anywhere!